pdc/apps/package/models.py

```diff
@@ -357,6 +359,37 @@ def export(self, fields=None):
         return result
 
 
+class ReleasedFiles(models.Model):
+    file_primary_key = models.IntegerField(blank=False, default=0)
+    repo             = models.ForeignKey(Repo)
+    released_date    = models.DateField(blank=True, null=True)
+    release_date     = models.DateField()
+    created_at       = models.DateTimeField(default=timezone.now)
+    updated_at       = models.DateTimeField(auto_now=True)
+    # 0 day release
+    zero_day_release = models.BooleanField(default=False)
+    obsolete         = models.BooleanField(default=False)
+
+    class Meta:
+        unique_together = (
+            ('file_primary_key', 'repo'),
+        )
+
+    def __unicode__(self):
+        return u"%s" % self.build
+
+    def export(self, fields=None):
+        _fields = (set(['file_primary_key', 'repo',
+                        'released_date', 'release_date', 'created_at',
+                        'updated_at', 'zero_day_release', 'obsolete'])
+                   if fields is None else set(fields))
+        result = model_to_dict(self, fields=_fields)
+        for k in ["release_date", "released_date", "created_at", "updated_at"]:
+            if k in result:
+                result[k] = str(result[k])
+        return result
```

Comment:
Yes, but you can see RPM and BuildImage are also do like this.
And the difference between these ones and others is that there is a "fields=None", this can let you choose the fields you want to show. 
I think this is better way, agree?

Reply:
Value of `fields` is not used anywhere. I don't like this is duplicated everywhere without being ever used. It makes the code unnecessarily complicated.

Reply:
When I init a field class in another serializer just list primary_key, released_data, that would be used. Just think this implement is good, so kept that part before.

Reply:
Usually I'm against the idea adding a code that could be potentially used later, because it won't be used in most cases :). If you think it's OK in this case, let's keep it.

Reply:
Yes, you are right, currently it has no use, but let's keep it, maybe we will use it :)